Header header

float64 start_t
float64 end_t

float64 tcl_length
float64 theta_yaw
float64 deltaTheta

float64[] qcl
float64[] tlc
float64[] q_x
